common/src/main/java/feast/common/logging/AuditLogger.java

```diff
@@ -113,28 +126,49 @@ private static void log(Level level, AuditLogEntry entry) {
       return;
     }
     if (entry.getKind().equals(AuditLogEntryKind.MESSAGE)
-        && !properties.isMessageLoggingEnabled()) {
+        && !properties.getMessageLogging().isEnabled()) {
       return;
     }
 
-    // Log event to audit log through enabled formats
-    String entryJSON = entry.toJSON();
-    switch (level) {
-      case TRACE:
-        log.trace(AUDIT_MARKER, entryJSON);
-        break;
-      case DEBUG:
-        log.debug(AUDIT_MARKER, entryJSON);
-        break;
-      case INFO:
-        log.info(AUDIT_MARKER, entryJSON);
-        break;
-      case WARN:
-        log.warn(AUDIT_MARKER, entryJSON);
-        break;
-      case ERROR:
-        log.error(AUDIT_MARKER, entryJSON);
-        break;
+    // Either forward log to logging layer or log to console
+    Map<String, Object> fluentdLogs = new HashMap<>();
```

Comment:
I suggest to replace this with `ImmutableMap` since there's no need for mutability and code would contain less of `fluentdLogs.put`